projects/entry-table/src/styles/modules/components/tables/entry-table-nested/rows/_general.scss

```diff
@@ -12,14 +12,6 @@
             .mat-cell, a {
                 color: map.value-from($theme, 'row.selected-color');
             }
-    
-            .mat-checkbox-checked .mat-checkbox-background {
-                background-color: map.value-from($theme, 'checkbox.background-color-selected');
```

Comment:
So we had "background-color-selected" property, we just had to set it in EBO?

Reply:
yes